modules/cbMap/processmap/importCSV.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+//ini_set('display_errors', 'on');
+require_once("config.inc.php");
+include_once('data/CRMEntity.php');
+include_once('modules/cbMap/cbMap.php');
+require_once('include/utils/utils.php');
+require_once('include/database/PearDatabase.php');
+
+global $adb, $log, $root_directory, $current_user;
+$current_user = new Users();
```

Comment:
Please change these two lines to:

```
$current_user = Users::getActiveAdminUser();
```

just in case we run into an install where the admin user 1 has been deleted